README.md

```diff
@@ -1,96 +1,117 @@
+
 ```swift
 import Numerics
 
-// All swift-numerics API is now available
+// The entire Swift Numerics API is now available
 ```
 
 Swift Numerics modules have minimal dependencies on other projects.
+
 The current modules assume only the availability of the Swift and C standard libraries and the runtime support provided by compiler-rt.
-Future expansion may assume the availability of other standard interfaces such as [BLAS (Basic Linear Algebra Subprograms)](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms) and [LAPACK (Linear Algebra Package)](https://en.wikipedia.org/wiki/LAPACK), but modules with more specialized dependencies (or dependencies that are not available on all platforms supported by Swift) belong in a separate package.
+
+Future expansion may assume the availability of other standard interfaces, such as [BLAS (Basic Linear Algebra Subprograms)](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms) and [LAPACK (Linear Algebra Package)](https://en.wikipedia.org/wiki/LAPACK). But modules with more specialized dependencies (or dependencies that are not available on all platforms supported by Swift) belong in a separate package.
 
 Because we intend to make it possible to adopt Swift Numerics modules in the standard library at some future point, Swift Numerics uses the same license and contribution guidelines as the Swift project.
 
 ## Using Swift Numerics in your project
-To use Swift Numerics in a SwiftPM project, add the following line to the dependencies in your `Package.swift` file:
+
+To use Swift Numerics in a SwiftPM project:
+
+1. Add the following line to the dependencies in your `Package.swift` file:
+
 ```swift
-.package(url: "https://github.com/apple/swift-numerics", from: "0.0.5"),
+.package(url: "https://github.com/apple/swift-numerics", from: "0.0.7"),
 ```
-add `Numerics` as a dependency for your target:
+
+2. Add `Numerics` as a dependency for your target:
+
 ```swift
 .target(name: "MyTarget", dependencies: [
   .product(name: "Numerics", package: "swift-numerics"),
   "AnotherModule"
 ]),
 ```
-and finally, add `import Numerics` in your source code.
-    
+
+3. Add `import Numerics` in your source code.
+
 ## Contributing to Swift Numerics
-Swift Numerics is a standalone library separate from the core Swift project.
-In practice, it will act as a staging ground for some APIs that may eventually be incorporated into the Swift Standard Library, and when that happens such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
+
+Swift Numerics is a standalone library that is separate from the core Swift project. In practice, it will act as a staging ground for some APIs that may eventually be incorporated into the Swift Standard Library. When that happens, such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
```

Comment:
```suggestion
Swift Numerics is a standalone library that is separate from the core Swift project, but it will sometimes act as a staging ground for APIs that will later be incorporated into the Swift Standard Library.
When that happens, such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
```

Reply:
Agree with the rewording and splitting into more paragraphs 👍 Will make these changes